packages/frint-react/src/components/Region.js

```diff
@@ -127,7 +130,17 @@ export default React.createClass({
       this.subscription.unsubscribe();
     }
 
-    // @TODO: clear instances
+    if (this.rootApp) {
+      this.state.listForRendering
+        .filter(item => item.multi === true)
```

Comment:
`multi` is a boolean, wouldn't it be the same to write `.filter(item => item.multi)` ?